process/tags/tagsCount.go

```diff
@@ -0,0 +1,115 @@
+package tags
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+)
+
+const (
+	tagsKey = "tags"
+)
+
+// CountTags defines what a TagCount handler should be able to do
+type CountTags interface {
+	Serialize() ([]*bytes.Buffer, error)
+	ParseTagsFromDB(response map[string]interface{}) error
+	ExtractTagsFromAttributes(attributes *data.Attributes)
+	GetTags() []string
+	Len() int
+}
+
+type tagsCount struct {
+	tags map[string]int
+}
+
+// NewTagsCount will create a new instance of tagsCount
+func NewTagsCount() CountTags {
+	return &tagsCount{
+		tags: make(map[string]int),
+	}
+}
+
+func (tc *tagsCount) Len() int {
+	return len(tc.tags)
+}
+
+func (tc *tagsCount) GetTags() []string {
+	tags := make([]string, 0, tc.Len())
+	for key := range tc.tags {
+		tags = append(tags, key)
+	}
+
+	return tags
```

Comment:
do we need the tags sorted? It is ok that they will arrive in any particular order?

Reply:
Tags don't need to be sorted. Just need to extract the tags.

Reply:
ok